pkg/cmd/factory/factory.go

```diff
@@ -112,6 +132,11 @@ func (c *ClientFlags) AddFlags(cmd *cobra.Command) {
 		"Private key path for TLS connection to CDC server")
 	cmd.PersistentFlags().StringVar(&c.logLevel, "log-level", "warn",
 		"log level (etc: debug|info|warn|error)")
+
+	cmd.PersistentFlags().StringVar(&c.User, "user", "", "User name for authentication. "+
```

Comment:
Why we call it ticdc user and ticdc password here? I think what we need is tidb's user and password, and we use it to verfiy the authorization. As a user, I think I will be confused about ticdc user name and tidb user name.

Reply:
The variable here refers to the authentication information provided to the cdc, so it is best not to be coupled to the underlying implementation. For example, in the future,authentication can be performed through a database that stores meta information.